Client/core/CMainMenu.cpp

```diff
@@ -382,18 +343,17 @@ void CMainMenu::SetMenuVerticalPosition(int iPosY)
     if (m_pHoveredItem)
     {
         m_unhoveredItems.insert(m_pHoveredItem);
-        m_pHoveredItem = NULL;
+        m_pHoveredItem = nullptr;
     }
 
     float fFirstItemSize = m_menuItems.front()->image->GetSize(false).fY;
     int   iMoveY = iPosY - m_menuItems.front()->image->GetPosition(false).fY - fFirstItemSize * 0.5f;
 
-    std::deque<sMenuItem*>::iterator it = m_menuItems.begin();
-    for (it; it != m_menuItems.end(); it++)
+    for (auto it : m_menuItems)
```

Comment:
Give it the actual type and a proper name.

Reply:
?